broker/cloud_functions/store_BigQuery/README.md

```diff
@@ -1,3 +1,3 @@
 # Store alerts in BigQuery
 
-This Cloud Function stores alert data in multiple BigQuery tables.
+This Cloud Function stores alert data in multiple BigQuery tables.
```

Comment:
This is not a real change. Try checking out the file again to cleanly restore the original version:

```
git checkout main/elasticc -- broker/cloud_functions/store_BigQuery/README.md
```